src/System.Data.Odbc/src/Configurations.props

```diff
@@ -1,13 +1,20 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
-    <BuildConfigurations>
+    <PackageConfigurations>
       netcoreapp2.0-FreeBSD;
```

Comment:
Unrelated to your change I do have to wonder why we have so many different configurations. It looks like we should only have a unix and windows configuration.

Reply:
OSX and Linux must be differentiated. The project uses $(TargetsOSX)

Reply:
We are using TargetsFreeBSD https://github.com/dotnet/corefx/blob/master/src/System.Data.Odbc/src/System.Data.Odbc.csproj#L116 and TargetsOSX as Dan said.

Reply:
Sorry I missed those :( I only did a quick scan. You can ignore my comment then.